### DISCUSSION TITLE
Baseline support?

### DISCUSSION THREAD

**Discussion by Flynn2018** [imaNNeo/fl_chart#1306] [General]
Does this plug-in support drawing a baseline on a bar chart? How should we do it? Thank you.

**Comment by imaNNeo** [comment]
Can you provide more detail about your question please?

**Reply by Flynn2018** [reply]
You can see orange baseline on the bar chart at  https://pic4.zhimg.com/v2-95c2430822bd7e8f4ef3f6caba7c7e33_b.jpg for example.

**Reply by imaNNeo** [reply]
Yes, you can use the `ExtraLines` feature.
Please check this issue: #476
